packages/Skeleton/AutoSkeletonizer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Text from '@tds/core-text'
+import Heading from '@tds/core-heading'
+import Image from '@tds/core-image'
+import Skeletons from './tds'
+import withSkeleton from './withSkeleton'
+
+const Skeletonize = children => {
```

Comment:
It would be best to keep the component name the same as the filename. So i think `Skeletonize` should be renamed to `AutoSkeletonizer`

Reply:
will do